### PR TITLE
docs: Add doc for all-in-one container bringup

### PR DESCRIPTION
**Pull request by ashokponkumar**
Signed-off-by: Ashok Pon Kumar <ashokponkumar@gmail.com>

### CODE REVIEW

File: docs/installation/web-interface.md [hunk 1]
```diff
@@ -8,11 +8,23 @@ nav_order: 2
 
 # Move2Kube Web Interface
 
+## Bringing up Move2Kube all-in-one container
+
+   ```console
+   $ mkdir -p workspace && cd workspace
+   $ docker run -p 8080:8080 -v $PWD:/workspace -v /var/run/docker.sock:/var/run/docker.sock -it quay.io/konveyor/move2kube-aio:latest
```
Comment (Akash-Nayak) [inline docs/installation/web-interface.md:15]: I am able to run the all-in-one container successfully and access the UI at `http://localhost:8080/` using these two commands. Then, I tried translating the cloud foundry sample usecase. After the translation was completed, when I tried to download the generated target artifacts, it downloaded a single json file. Tried doing the translation for the language platforms sample and for that also a json file was downloaded when I clicked on `Get`.

Screenshot 2020-12-28 at 7 49 37 PM

Comment (ashokponkumar) [reply docs/installation/web-interface.md:15]: Can you try with chrome once? And also in the current one can you check the url pointed to by get?

Comment (Akash-Nayak) [reply docs/installation/web-interface.md:15]: Yes, I tried with chrome it is working fine and I am able to download the zip file containing the generated artifacts. Hovering over the `Get` does not show the url, in the browser console the filename is `/api/v1/applications/newapp/targetartifacts/newapp_1609164843`.

Screenshot 2020-12-28 at 8 15 16 PM

Comment (ashokponkumar) [reply docs/installation/web-interface.md:15]: Then it's some firefox compatibility issue. Let's merge this and create a new issue in move2kube-ui? Btw, did the docker-compose method work in Firefox?

Comment (Akash-Nayak) [reply docs/installation/web-interface.md:15]: Yes sure. I just checked with docker-compose method also, I am facing the same issue with that as well.

**Review by Akash-Nayak — APPROVED** [review]

**Pull request merged**